views/partials/_banner.njk

```diff
@@ -1,47 +1,36 @@
-{% from "govuk/components/phase-banner/macro.njk" import govukPhaseBanner %}
-{% set phaseBannerClasses = "app-phase-banner app-width-container" %}
-{% if masthead %}
-  {% set phaseBannerClasses = phaseBannerClasses + " app-phase-banner--no-border" %}
-{% endif %}
-
-<div class="app-phase-banner__wrapper">
 {% if PULL_REQUEST === "true" %}
-  {% set phaseBannerText %}
+  {% set bannerTag = "preview" %}
+  {% set bannerText %}
     This is a preview of
     a <a class="govuk-link" href="https://github.com/alphagov/govuk-design-system/pull/{{ REVIEW_ID  }}">proposed change</a> to the
     <a class="govuk-link" href="https://design-system.service.gov.uk">Design System</a>.
   {% endset %}
-
-  {{ govukPhaseBanner({
-    "tag": {
-      "text": "preview",
-      "classes": "app-tag--review"
-    },
-    "classes": phaseBannerClasses,
-    "html": phaseBannerText
-  }) }}
 {% elif PULL_REQUEST === "false" %}
-  {% set phaseBannerText %}
+  {% set bannerTag = "archive" %}
+  {% set bannerText %}
     This is an archived version of
     the <a class="govuk-link" href="https://github.com/alphagov/govuk-design-system/tree/{{ BRANCH }}">{{ BRANCH }}</a> branch for the
     <a class="govuk-link" href="https://design-system.service.gov.uk">Design System</a>.
   {% endset %}
+{% endif %}
+
+{% if bannerText %}
+  {% from "govuk/components/phase-banner/macro.njk" import govukPhaseBanner %}
+
+  <div class="app-phase-banner__wrapper">
```

Comment:
Only semi-related to this change - do we think we can remove this? It's only used to exclude the phase banner from the axe tests - I wonder if we can target `govuk-phase-banner` instead?

Reply:
Good catch – I'll push a commit to do just that.